tests/e2e/specs/backend/product-query.test.ts

```diff
@@ -44,7 +44,12 @@ describeOrSkip( GUTENBERG_EDITOR_CONTEXT === 'gutenberg' )(
 			await expect( page ).toRenderBlock( block );
 		} );
 
-		it( 'Editor preview shows only on sale products after enabling `Show only products on sale`', async () => {
+		/**
+		 * we changed the show only products on sale from a top-level toggle
+		 * setting to a product filter but tests for them haven't been updated
+		 * yet. we will fix these tests in a follow-up pr.
```

Comment:
```suggestion
		 * We changed the “Show only products on sale” from a top-level toggle
		 * setting to a product filter, but tests for them haven't been updated
		 * yet. We will fix these tests in a follow-up PR.
```

Sorry, grammar nazi time 😬